### ISSUE TITLE
Handle new Interval type

### ISSUE THREAD

**Issue by tebeka** [quantopian/qgrid#113]
Pandas 0.20.0 introduced [Interval type](http://pandas.pydata.org/pandas-docs/stable/whatsnew.html#whatsnew-0200-enhancements-intervalindex). Currently qgrid raises 'TypeError' when handling it.

Here's an example code to reproduce the problem:
```python
import numpy as np
import pandas as pd

td = np.cumsum(np.random.randint(1, 15*60, 1000))
start = pd.Timestamp('2017-04-17')
df = pd.DataFrame(
    [(start + pd.Timedelta(seconds=d)) for d in td],
    columns=['time'])

freq = '15Min'
start = df['time'].min().floor(freq)
end = df['time'].max().ceil(freq)
bins = pd.date_range(start, end, freq=freq)

df['time_bin'] = pd.cut(df['time'], bins)

import qgrid
qgrid.show_grid(df)
```

**Comment by TimShawver** [comment]
Thanks, haven't looked into this yet but I should be able to fix it in the next release.

**Comment by TimShawver** [comment]
This is mostly fixed in qgrid 1.0, which is currently available in beta form.  See the installation instructions on the [readme page](https://github.com/quantopian/qgrid#installation).

I say "mostly" fixed because I didn't add special support for sorting/filtering/editing interval columns.  The way I handle them is I basically just convert the values to strings, and treat it as a string column. I realize this isn't ideal but I think it's all the time I can afford to spend on Interval columns right now, since they're complicated to deal with and I assume that they're not all that common since they're new.
